wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -71,15 +77,15 @@ workflow PhysicalAndStatisticalPhasing {
         }
 
         call H.SplitVCFbySample as SplitVcfbySampleSV { input:
-            joint_vcf = SubsetVcfSV.subset_vcf,
+            joint_vcf = UnphaseSVGenotypes.filter_phasing_vcf,
             region = region,
             samplename = sample_id
         }
 
         call ConvertLowerCase {
             input:
                 vcf = SplitVcfbySampleSV.single_sample_vcf,
-                samplename = sample_id
+                samplename = sample_id + ".uppercased"
```

Comment:
Right, looking at everything again, perhaps `samplename` -> `prefix` here and below and remove all `_sv_cleaned` below.